util/execdetails/execdetails.go

```diff
@@ -322,6 +364,18 @@ func (e *RuntimeStatsColl) ExistsCopStats(planID string) bool {
 	return exists
 }
 
+// GetReaderStats gets the ReaderRuntimeStats specified by planID.
+func (e *RuntimeStatsColl) GetReaderStats(planID string) *ReaderRuntimeStats {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+	stats, exists := e.readerStats[planID]
+	if !exists {
+		stats = &ReaderRuntimeStats{copRespTime: make([]time.Duration, 0, 20)}
```

Comment:
Just curious, why setting the slice capacity to `20`?

Reply:
Just a casual value.

Reply:
Do you have any idea or just set it zero.